src/logic/config/utils/index.ts

```diff
@@ -0,0 +1,11 @@
+import { setNetworkId, getNetworkName, getConfig } from 'src/config'
+import { ETHEREUM_NETWORK } from 'src/config/networks/network'
+import { makeNetworkConfig } from 'src/logic/config/model/networkConfig'
+import { configStore } from 'src/logic/config/store/actions'
+import { store } from 'src/store'
+
+export const setNetwork = (id: ETHEREUM_NETWORK) => {
+  setNetworkId(getNetworkName(id))
+  const safeConfig = makeNetworkConfig(getConfig())
+  store.dispatch(configStore(safeConfig))
+}
```

Comment:
@dasanra I think you were planning to do it via Redux?

Reply:
I abstracted this from four positions. If we want to add it to Redux, it would be nice if @dasanra could lend a hand.